index.bs

```diff
@@ -1245,7 +1246,7 @@ Technical Architecture Group Appointments</h5>
 	(Partial terms used to fill a vacated seat do not count towards this limit.)
 
 	Note: Individuals who have reached the limit of two consecutive appointed terms
-	<em class=rfc2119>may</em> freely run for election
+	can freely run for election
```

Comment:
I think this would make sense with the MAY wording, i.e., not as a Note (and added to the preceding paragraph).

Reply:
Perhaps it should explain why no permission ("may") is needed to run for election – there are no limits on terms for those elected. "Since there are no limits on elected terms, this limit on appointed terms does not constrain these individuals from running for election at any time." ?